api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsCurrentlyOnArtWithoutTbScreening.java

```diff
@@ -0,0 +1,119 @@
+package org.openmrs.module.eptsreports.reporting.reports;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+import org.openmrs.Location;
+import org.openmrs.module.eptsreports.reporting.library.datasets.DatimCodeDatasetDefinition;
+import org.openmrs.module.eptsreports.reporting.library.datasets.ListOfPatientsCurrentlyOnArtWithoutTbScreeningDataset;
+import org.openmrs.module.eptsreports.reporting.library.datasets.SismaCodeDatasetDefinition;
+import org.openmrs.module.eptsreports.reporting.library.datasets.TotalOfPatientsCurrentlyOnArtWithoutTbScreeningDataSet;
+import org.openmrs.module.eptsreports.reporting.reports.manager.EptsDataExportManager;
+import org.openmrs.module.reporting.ReportingException;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SetupListOfPatientsCurrentlyOnArtWithoutTbScreening extends EptsDataExportManager {
```

Comment:
No any base cohort is needed for this report to limit number of patients returned?

Reply:
For this report all patients are picked from tx-curr and tx-curr already implemented the basic base cohort. I dont know if in this scenario there is a need to include the basic base cohort in the setup